frog/imports/api/sessions.js

```diff
@@ -85,11 +85,18 @@ Meteor.methods({
   },
   'add.session': graphId => {
     const sessionId = uuid();
-    const copyGraphId = addGraph('SESSION_GRAPH<' + sessionId + '>');
+    const graph = Graphs.findOne(graphId);
+    const graphs = Graphs.find({}).fetch();
```

Comment:
Not efficient to download all the graphs object from the database to then check all of their names. My next commit suggest a nicer way to do the same thing